### PR TITLE
WIP : GUI: add default button

### LINKED ISSUE #208: GUI: Add Reset to Defaults button
This would allow user to reset the settings back to defaults

>change the settings dictionary in `GUI.py`

### PR DESCRIPTION
**Pull request by lohani2280**
fixes #208

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

File: uiplib/GUI.py [hunk 1]
```diff
@@ -51,7 +52,7 @@ def __init__(self, settings):
         self.root = Tk()
         self.root.resizable(width=False, height=False)
         # set window title
-        self.root.title("UIP")
+        self.root.title("WIN")
```
Comment (nemaniarjun) [inline uiplib/GUI.py]: Why this change?

Comment (lohani2280) [inline uiplib/GUI.py]: oops , its by mistake..i'll change it
kindly review other things

File: uiplib/GUI.py [hunk 2]
```diff
@@ -126,6 +127,13 @@ def create_settings_tab(self):
                               command=self.handle_settings)
         apply_button.grid(row=6, column=1, pady=20, sticky=W)
 
+         # Reset
+        default_button = Button(mainFrame,
```
Comment (abhsag24) [inline uiplib/GUI.py]: needs to be in the settings tab please

File: uiplib/GUI.py [hunk 3]
```diff
@@ -274,6 +282,15 @@ def handle_settings(self):
             return
         update_settings(self.new_settings)
 
+    def default_settings(self):
+        self.new_settings['pics-folder'] = DEFAULT_SETTINGS['pics-folder']
```
Comment (abhsag24) [inline uiplib/GUI.py]: you can extract default settings from `setup.py`

Comment (abhsag24) [inline uiplib/GUI.py]: you can just do self.new_settings = DEFAULT_SETTINGS or something like that?

Comment (lohani2280) [inline uiplib/GUI.py]: yeah , sure i'll edit it
